src/App/App.js

```diff
@@ -21,25 +22,38 @@ class App extends Component {
         })
         .catch(() => this.setState({ ...this.state, error: true }))
     }
-   componentDidMount() {
-      this.grabAPI()
-    }
+
+  componentDidMount() {
+    this.grabAPI()
+  }
     
+    grabSingleMovie = (id) => {
+      fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+        .then(response => response.json())
+        .then(data => {
+            this.setState({ ...this.state, currentMovie: [data.movie] })
+        })
+        .catch(() => this.setState({ ...this.state, error: true }))
+    }
+
     displaySingleMovie = (id) => {
-      const findMovie = this.state.movies.find(movie => movie.id === id);
-      console.log(findMovie)
-      this.setState({...this.state, movies: [findMovie]})
+      this.grabSingleMovie(id)
+      this.setState({ ...this.state, movies: [] })
+      console.log('movie', this.state.currentMovie);
+    }
+
+    goHome = () => {
+      this.grabAPI()
+      this.setState({ ...this.state, currentMovie: null })
     }
-    
-   goHome = () => {
-    this.grabAPI()
-   }
 
   render() {
     return (
       <section >
         <Nav goHome={this.goHome}/>
-        <MovieContainer movies={this.state.movies} singleMovie={this.displaySingleMovie}/>
+        {!this.state.currentMovie ? 
```

Comment:
This came in clutch! its awesome how you were able to implement ternary here to get it to work!